apps/oauth_callback_broker.py

```
@@ -0,0 +1,653 @@
+            client=flow.client,
+            provider=flow.provider,
+            code=code,
+            redirect_uri=redirect_uri,
+        )
+
+        if flow.retry_policy is None:
+            result = flow.on_authorized()
+            if result is False:
+                raise RuntimeError(
+                    f"OAuth flow {flow_ref!r} remained unavailable after authorization",
+                )
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)